src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double upperWidth;
+    private double lowerWidth;
+    private double height;
+
+    public IsoscelesTrapezoid(Color color, double upperWidth, double lowerWidth, double height) {
+        super(color);
+        this.upperWidth = upperWidth;
+        this.lowerWidth = lowerWidth;
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
```

Comment:
Method names must be verbs
```suggestion
    public double calculateArea() {
```